src/routes/(admin)/questionbank/+page.svelte

```
@@ -14,9 +14,28 @@
 	$effect(() => useSidebar().setOpen(true));
 	const { data } = $props();
 	let currentRow: number | null = $state(null);
+	let filteredTags: string[] = $state([]);
+	let filteredStates: string[] = $state([]);
+	let filteredSearch: string = $state('');
+	let filteredQuestions = $derived.by(() => {
+		return filteredTags.length === 0 && filteredStates.length === 0 && filteredSearch.length === 0
+			? data && data?.questions
+			: data &&
+					data?.questions.filter(
+						(question: any) =>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)